egs/aidatatang_200zh/s5/local/download_and_untar.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
```

Comment:
authors + license (Apache 2.0)

Reply:
Thank you. I have completed the copyright part, please check it.